### ISSUE TITLE
Y-axis does not scale to very small values

### ISSUE THREAD

**Issue by candlerb** [prometheus/prometheus#3084]
**What did you do?**

With default install and default scraping config, browse the `scrape_duration_seconds` metric

**What did you expect to see?**

Nicely labelled graph

**What did you see instead? Under which circumstances?**

The Y axis labels are "0.00", "0.00", "0.00", "0.00"

![image](https://user-images.githubusercontent.com/44789/29372971-9e48beae-82a4-11e7-9058-1db1130d9a2e.png)

Checking some data points, the labels here should be "0.001", "0.002", "0.003" and "0.004" (the largest data point is 0.00396 and is just below the 0.004 line)

Or perhaps these would be better rendered as 1.0m, 2.0m, 3.0m, 4.0m

**Environment**

* System information:

        Linux 4.4.0-87-generic x86_64

* Prometheus version: 

    ~~~
    prometheus, version 2.0.0-beta.1 (branch: HEAD, revision: 4dcb465029f39a2a3030333a81753f8a40ec90ce)
      build user:       root@d3f9974fac5a
      build date:       20170811-12:12:36
      go version:       go1.8.3
    ~~~

* Alertmanager version:

    n/a

* Prometheus configuration file:

    unmodified default

* Alertmanager configuration file:

    n/a

* Logs:

    n/a

**Comment by Gouthamve** [comment]
@yamatoya @grobie @beorn7 Can this be closed ref: #3184 ?

**Comment by beorn7** [comment]
Yes, this should be addressed in 1.7.2. @candlerb please re-open if that's not the case.
